app/src/main/java/org/pesmypetcare/mypetcare/activities/threads/GetPetImageRunnable.java

```diff
@@ -0,0 +1,37 @@
+package org.pesmypetcare.mypetcare.activities.threads;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+
+import org.pesmypetcare.mypetcare.activities.MainActivity;
+import org.pesmypetcare.mypetcare.utilities.ImageManager;
+
+import java.io.IOException;
+
+public class GetPetImageRunnable implements Runnable {
+    private int actual;
+    private String username;
+    private String petName;
+    private Bitmap defaultBitmap;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'defaultBitmap' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5421176)